Server/MobileDevice.java

```diff
@@ -0,0 +1,9 @@
+public class MobileDevice extends Device {
+    private List<Notification> notification_list;
+
+    public MobileDevice(List_notification notification_list) 
```

Comment:
Αρχικά το notification_list είναι σίγουρα άδειο, άρα δε χρειάζεται να το περάσουμε ως παράμετρο στον constructor. Απλά δημιουργούμε το αντικείμενο με 

`this.notification_list = new ArrayList<Notification>();`

μέσα στον constructor ή με

`private List<notification> notification_list = new ArrayList<>();`

στη δήλωση της μεταβλητής.